xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/cssVariablesInit.css

```diff
@@ -37,13 +37,15 @@
 ## List of versions where variables were introduced:
 ## * 17.3.0RC1: 100vw, int-viewport-width, font-weight-regular, font-weight-semibold, font-weight-bold
 ## * 17.9.0RC1: login-form-max-width
+## * 18.0.0RC1: mark-bg
 #set($newVariables = {
 "100vw": "100vw",
 "int-viewport-width": "calc(10000 * tan(atan2(var(--100vw), 10000px)))",
 "login-form-max-width": "450px",
 "font-weight-regular": 400,
 "font-weight-semibold": 700,
-"font-weight-bold": "900"
+"font-weight-bold": "900",
+"mark-bg": "color-mix(in srgb, var(--btn-warning-bg), white)"
```

Comment:
I think it is dangerous to mix colors this way. How do you know that mixing this color with white will give you a color that has enough contrast with the text color? What about making this an actual color theme preference that defaults to this mix?